sdk/core/azure-core/src/test/java/com/azure/core/util/logging/ClientLoggerTests.java

```diff
@@ -276,6 +277,18 @@ public void logExceptionAsErrorStackTrace() {
         assertTrue(logValues.contains(runtimeException.getStackTrace()[0].toString()));
     }
 
+    @Test
```

Comment:
Let's just make this a parameterized test that accepts an `int logLevelToConfigure`, `LogLevel logLevelToValidate`, and `boolean expected` to get better coverage on this.